.github/workflows/ci.yml

```diff
@@ -206,9 +207,15 @@ jobs:
         run: |
           set -euxo pipefail
           wget -qO- https://apt.llvm.org/llvm-snapshot.gpg.key | sudo tee /etc/apt/trusted.gpg.d/apt.llvm.org.asc
-          echo deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy main | sudo tee /etc/apt/sources.list.d/llvm.list
+          echo deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-18 main | sudo tee /etc/apt/sources.list.d/llvm.list
           sudo apt update
-          sudo apt -y install clang gcc-multilib llvm locate qemu-system-{arm,x86}
+          sudo apt -y install clang-18 gcc-multilib llvm-18 locate qemu-system-{arm,x86}
+
+      - name: Add LLVM to PATH
+        if: runner.os == 'Linux'
+        run: |
+          set -euxo pipefail
+          echo /usr/lib/llvm-${{ env.LLVM_VERSION }}/bin >> $GITHUB_PATH
```

Comment:
is there any reason not to put this in the previous "install prerequisites" step?

also, let's either remove `LLVM_VERSION` or use it everywhere (there are two more instances of `18`)